roles/user_models.py

```diff
@@ -0,0 +1,29 @@
+"""
+User data models (mostly used for the Rest Framework)
+"""
+from __future__ import unicode_literals
+
+
+class UserGroup(object):
+    """
+    Class defining the relationship between an user name and a basic group type
+    """
+    def __init__(self, username, group_type):
+        self.username = username
+        self.group_type = group_type
+
+    def __repr__(self):
+        return "<username:{username} | group_type:{group_type}>".format(
+            username=self.username,
+            group_type=self.group_type
+        )
+
+    def __eq__(self, other):
+        try:
```

Comment:
Explicit is better than implicit. There could be other differences unforeseen, and it would no longer be obvious which attributes we care about. I also think this use of `except AttributeError` is good, and not to be avoided.

It _could_ be good to add a check that `other` is also a `UserGroup`, but that could be overkill.